webpack.config.ts

```diff
@@ -49,14 +54,7 @@ const config = (env: EnvConfig): Configuration => {
                 },
                 {
                     test: /\.(png|woff|woff2|eot|ttf|svg)$/,
-                    use: [
-                        {
-                            loader: 'file-loader',
-                            options: {
-                                name: 'assets/[name].[hash].[ext]',
-                            },
-                        },
-                    ],
```

Comment:
Since we are no longer using `file-loader` do we still need this package? If not lets clean it up

Reply:
If we use the same asset in multiple places with the loader as `asset/inline`, will it increase the bundle size?

Reply:
> Since we are no longer using `file-loader` do we still need this package? If not lets clean it up

yeah, we don't need it anymore since we are suing webpack asset module now. will remove it. thanks :pray:

Reply:
> If we use the same asset in multiple places with the loader as `asset/inline`, will it increase the bundle size?

based on webpack 5 docs we should use `Asset Module`s, `asset/inline` is kinda the same as `url-loader`,  as mentioned in [webpack 5 asset module docs](https://webpack.js.org/guides/asset-modules/) :

> asset/inline exports a data URI of the asset. Previously achievable by using url-loader.

so I don't think it affects the bundle size on reusing the same asset in different places.